.changeset/warm-pigs-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/text': major
```

Comment:
Hey @commercetools/shield-team-fe 

This should have been a `minor` bump, not `major`.

Please be careful especially about `major` bumps, as it usually imply breaking changes.

I'll update the changeset.

Reply:
Oh, sorry didn't know that... Noted for the future and thank you